CRADLE/Normalize/normalize.py

```diff
@@ -112,20 +112,31 @@ def excludeOverlapRegion():
 		if len(idx) > 0:
 			continue
 
-		## overlap Case 2
+		## overlap Case 2: Overlap regions that only end inside the region
 		idx = np.where(
 			(overlapREGION[:,0] == regionChromo) &
+			(overlapREGION[:,1].astype(int) < regionStart) &
 			(overlapREGION[:,2].astype(int) > regionStart) &
-			(overlapREGION[:,2].astype(int) <= regionEnd)
+			(overlapREGION[:,2].astype(int) < regionEnd)
 			)[0]
 		if len(idx) > 0:
 			overlapThis.extend( overlapREGION[idx].tolist() )
 
-		## overlap Case 3
+		## overlap Case 3: Overlap regions that only start inside the region
```

Comment:
Could this case happen? `overlapREGION` should be the subset of regions and I wonder if this can happen at all?

Reply:
I think it's useful to be thorough. Even if it can't happen, proving that is the case on all possible inputs is probably more trouble than it's worth.